web/cypress/support/commands.ts

```diff
@@ -141,12 +145,14 @@ Cypress.Commands.add('createTestByName', (name: string) => {
   cy.get('[data-cy=test-details-name]').should('have.text', `${name} (v1)`);
 });
 
-Cypress.Commands.add('editTestByTestId', (testId: string) => {
-  cy.get(`[data-cy=test-actions-button-${testId}]`).click();
-  cy.get('[data-cy=test-card-edit]').click();
+Cypress.Commands.add('editTestByTestId', () => {
+  cy.interceptEditTestCall();
   cy.get('[data-cy=edit-test-form]').should('be.visible');
   cy.get('[data-cy=create-test-name-input] input').clear().type('Edited Test');
   cy.get('[data-cy=edit-test-submit]').click();
+  cy.wait('@testEdit');
```

Comment:
Waiting for test edit and fetching the run

Reply:
🔥